webapp/src/tutorial.tsx

```diff
@@ -433,12 +435,20 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         }
     }
 
+    private handleResize() {
+        const options = this.props.parent.state.tutorialOptions;
+        this.setShowSeeMore(options.autoexpandStep);
+    }
+
     componentDidMount() {
         this.setShowSeeMore(this.props.parent.state.tutorialOptions.autoexpandStep);
+        this.resizeDebouncer = pxt.Util.debounce(this.handleResize, 500);
+        window.addEventListener('resize', this.resizeDebouncer);
     }
 
     onMarkdownDidRender() {
         this.setShowSeeMore(this.props.parent.state.tutorialOptions.autoexpandStep);
+        window.removeEventListener('resize', this.resizeDebouncer);
```

Comment:
Should this one be in `componentWillUnmount`? Removing the event listener in `onMarkdownDidRender` seems like it wouldn't fix the issue as it would stop resizing when the page has loaded

Reply:
oh yeah you're right thanks joey! It was rudely working, fooling me!